save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -74,17 +75,16 @@ internal class OrganizationController(
         organizationService.findAll()
     }
 
-    @GetMapping("/not-deleted")
+    @PostMapping("/not-deleted")
     @PreAuthorize("permitAll()")
     @Operation(
-        method = "GET",
+        method = "POST",
         summary = "Get non-deleted organizations.",
         description = "Get non-deleted organizations.",
     )
     @ApiResponse(responseCode = "200", description = "Successfully fetched non-deleted projects.")
-    fun getNotDeletedOrganizations() = Mono.fromCallable {
-        organizationService.getNotDeletedOrganizations()
-    }
+    fun getNotDeletedOrganizations(@RequestBody(required = false) organizationFilters: OrganizationFilters?) =
+            organizationService.getNotDeletedOrganizations(organizationFilters?.name).toFlux()
```

Comment:
Same here, its better to pass filters into service

Reply:
Fixed